pkg/apis/kudo/v1beta1/operatorversion_types.go

```diff
@@ -95,6 +109,9 @@ type Parameter struct {
 	// Trigger identifies the plan that gets executed when this parameter changes in the Instance object.
 	// Default is `update` if a plan with that name exists, otherwise it's `deploy`.
 	Trigger string `json:"trigger,omitempty"`
+
+	// Type specifies the value type. Defaults to `string`.
+	Type ParameterType `json:"value-type,omitempty"`
```

Comment:
We can't use `type` here, because this will result in bugs with Kubernetes' schema validation. This bug didn't show when testing locally against a Kubernetes 1.17.2 cluster, so it might already be fixed. Anyways, the version of `control-plane` used in the integration tests is affected, other versions possibly as well.

Reply:
I think it's the combination of `type: array` triggering this, as this is also used in JSON Schema. The test was working with `type: list`. But we also use `type: string` which might trigger bugs as well.